lib/email-delivery.js

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var P = require('./promise')
+
+module.exports = function (log) {
+
+  return function start(deliveryQueue) {
+
+    function getHeaderValue(headerName, message){
```

Comment:
Is this exactly the same as the one in the bouncer script? Should we extract it to a common module and `require` it in?